index.html

```diff
@@ -15,17 +15,40 @@ <h1>Tanja Süß</h1>
       <a href="#biography">Biography</a>
     </nav>
   </header>
-  <body>
     <figure> 
-      <img src="https://source.unsplash.com/random/600x300" alt="random picture">
+      <img src="https://source.unsplash.com/random/600x300" alt="">
       <figcaption>Picture from Unsplash randomly chosen</figcaption>
    </figure>  
+   <ul>Soft Skills
+     <li>love to learn</li>
+     <li>curiosity</li>
+     <li>positive energy</li>
+     <li>team player</li>
+     <li>endurance</li>
+   </ul>
+   <ul>IT Skills
+     <li>Git/GitHub</li>
+     <li>HTML</li>
+     <li>Accessibility in Programming</li>
+     <li>CSS</li>
+     <li>Javascript</li>
+     <li>...to be continued</li>
```

Comment:
Great skills! 
I am not sure but maybe these lists could be nested in a separate section?

Reply:
Many thanks for your review - this will be a question for our head-coach i think 😃